dotnet/src/Experimental/Agents/Internal/ChatThread.cs

```diff
@@ -50,33 +50,29 @@ public static async Task<IAgentThread> GetAsync(OpenAIRestContext restContext, s
     }
 
     /// <inheritdoc/>
-    public async Task<IChatMessage> AddUserMessageAsync(string message, CancellationToken cancellationToken = default)
+    public async Task<IChatMessage> AddUserMessageAsync(string message, IEnumerable<string>? fileIds = null, CancellationToken cancellationToken = default)
     {
         this.ThrowIfDeleted();
 
-        var messageModel =
-            await this._restContext.CreateUserTextMessageAsync(
-                this.Id,
-                message,
-                cancellationToken).ConfigureAwait(false);
+        var messagemodel = await this._restContext.CreateUserTextMessageAsync(this.Id, message, fileIds: null, cancellationToken).ConfigureAwait(false);
 
-        return new ChatMessage(messageModel);
+        return new ChatMessage(messagemodel);
```

Comment:
Nit: why did you change the nameCasing to be incorrect (for C#) now?

Reply:
As you may be familiar, sometimes visual studio does things. ;)